### PR TITLE
Update Core CHANGELOG for M59

### PR DESCRIPTION
**Pull request by morganchen12**

### CODE REVIEW

File: Firebase/Core/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# v6.3.3 -- M59
+- [changed] Carthage and zip file distributions are now built with Xcode 11.
```
Comment (maksymmalyhin) [inline Firebase/Core/CHANGELOG.md]: nit: Maybe we should be specific and specify Xcode 11.0 version just to clear up any confusion? @paulb777 WDYT?

Comment (paulb777) [reply Firebase/Core/CHANGELOG.md]: Definitely we should exactly specify the Xcode version.

Comment (paulb777) [inline Firebase/Core/CHANGELOG.md]: Add a sentence that Xcode 10.x is no longer supported with this release from the zip file and Carthage.

Comment (morganchen12) [reply Firebase/Core/CHANGELOG.md]: Done (internal CL updated as well).

Comment (morganchen12) [reply Firebase/Core/CHANGELOG.md]: Done, updated internal CL as well.

**Pull request merged**